api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -493,15 +488,26 @@ private User initializeUserIfNeeded() {
   @Override
   public ResponseEntity<Profile> syncTrainingStatus() {
     User user = userProvider.get();
-    Profile profile = profileService.getProfile(user);
     try {
       userService.syncUserTraining(user);
     } catch(NotFoundException ex) {
       throw ex;
     } catch (ApiException e) {
       throw new ServerErrorException(e);
     }
-    return ResponseEntity.ok(profile);
+    return getProfileResponse(user);
+  }
+
+  @Override
+  public ResponseEntity<Profile> syncEraCommonsStatus() {
+    User user = userProvider.get();
+    NihStatus nihStatus = fireCloudService.getNihStatus();
```

Comment:
Question: Can user linked nih username change? I guess i am trying to think if it will be usefull to check if the username already exist in DB and set it accordingly. Thoughts?

Reply:
But since we are not making a special call to get the username i think its fine

Reply:
I don't believe it can change, but I am unsure...? This is meant to update with what is in FC though, so I'm inclined to do a full update.

Reply:
FYI, linked username **CAN** change. It is not a thing that is within our control, and we should not be trying to enforce restrictions there.